src/content/docs/apm/agents/php-agent/php-agent-api/newrelic_set_error_group_callback.mdx

```diff
@@ -0,0 +1,238 @@
+---
+title: newrelic_set_error_group_callback (PHP agent API)
+type: apiDoc
+shortDescription: This method allows a user to define a custom callback function to generate error group names.
+tags:
+  - Agents
+  - PHP agent
+  - PHP agent API
+metaDescription: 'Use this API to register a callback which returns a custom error group name.'
+redirects:
+  - /docs/agents/php-agent/php-agent-api/newrelic_set_error_group_callback
+---
+
+Group errors with your own custom-defined fingerprinting callback function.
+
+## Syntax
+
+```php
+newrelic_set_error_group_callback($callback)
+```
+
```

Comment:
Documentation for other APIs have the syntax first and the description right below the syntax. For consistency's sake, I suggest to update docs for these two new APIs to match others.

Reply:
Fixed in 2689de83d601ddb329e70f1170f4a2a04c26675a